src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -48,54 +50,32 @@
       :accessibility-label :send-label}
      (i18n/label :t/to)]
     [quo/summary-tag
-     {:label               "Mark Libot"
-      :type                :user
-      :image-source        (resources/get-mock-image :user-picture-male4)
-      :customization-color :magenta}]]])
+     {:type  :address
+      :label (utils/get-shortened-address to-address)}]]])
 
-(defn- transaction-from
-  [status-account-props theme]
+(defn- user-summary
+  [{:keys [amount account-props theme label accessibility-label summary-type]}]
   [rn/view
    {:style {:padding-horizontal 20
             :padding-bottom     16}}
    [quo/text
     {:size                :paragraph-2
      :weight              :medium
      :style               (style/section-label theme)
-     :accessibility-label :summary-from-label}
-    (i18n/label :t/from-capitalized)]
+     :accessibility-label accessibility-label}
+    label]
    [quo/summary-info
-    {:type          :status-account
+    {:type          summary-type
      :networks?     true
-     :values        {:ethereum 150
-                     :optimism 50
-                     :arbitrum 25}
-     :account-props status-account-props}]])
-
-(defn- transaction-to
-  [user-props theme]
-  [rn/view
-   {:style {:padding-horizontal 20
-            :padding-bottom     16}}
-   [quo/text
-    {:size                :paragraph-2
-     :weight              :medium
-     :style               (style/section-label theme)
-     :accessibility-label :summary-from-label}
-    (i18n/label :t/to-capitalized)]
-   [quo/summary-info
-    {:type          :user
-     :networks?     true
-     :values        {:ethereum 150
-                     :optimism 50
-                     :arbitrum 25}
-     :account-props user-props}]])
+     :values        {:ethereum amount
+                     :optimism 0
+                     :arbitrum 0}
```

Comment:
Are we setting the remaining networks as zero because we are not supporting them rn?
If so, maybe we can remove them or add a comment specifying this is temporal.

Reply:
I need to remove this and this should be pulled from data 👍